File: Backend/src/main/java/com/example/proiecttw/hospital/dto/LoginResponse.java
package com.example.proiecttw.hospital.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class LoginResponse {
    private Long id;
    private String role; // "PATIENT", "DOCTOR", "ADMIN"
    private String username;
    private String firstName;
    private String lastName;
    private String message;
}

File: Frontend/Android App/app/src/main/java/com/example/proiecttw_android/data/api/models/LoginResponse.kt
package com.example.proiecttw_android.data.models

data class LoginResponse(
    val id: Long? = null,
    val role: String? = null,      // "PATIENT", "DOCTOR"
    val username: String? = null,
    val firstName: String? = null,
    val lastName: String? = null,
    val message: String? = null
)
